DEMO.md

```diff
@@ -104,6 +104,10 @@ the response should look like
 
 Keep track of the `endpoint`, it will be used later when pushing an event to this ingress application.
 
+```bash
+export BRIDGE_ENDPOINT=http://ob-87508471-ee0f-4f53-b574-da8a61285986-ob-kekkobar.apps.openbridge-dev.fdvn.p1.openshiftapps.com
```

Comment:
@jiripetrlik Can you use jq to parse curl response and store the endpoint? That way users don't have to edit your command, just execute it.

Reply:
@sutaakar Thank you for good tip. Done.